spec/models/searchgov_url_spec.rb

```diff
@@ -313,6 +388,14 @@
           end
         end
       end
+
+      context 'when destroying' do
+        it 'deletes the searchgov_document' do
+          fetch
+          expect { searchgov_url.destroy }.to change { SearchgovDocument.count }.
+            from(1).to(0)
+        end
+      end
```

Comment:
This spec can be removed, as it will be covered by the association spec I mentioned above:
```ruby
it { is_expected.to have_one(:searchgov_document).dependent(:destroy) }
```